File: python/ram.py
class ram:

    __capacity = ''
    __price = 0

    def __init__(self, capacity = '', price = 0):
        self.__capacity = capacity
        self.__price = price
    
    #prosedur setter
    def setCapacity(self, capacity):
        self.__capacity = capacity
    
    def setPrice(self, price):
        self.__price = price
    
    #Prosedur getter
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price
    
    

File: PHP/ram.php
<?php

    class ram
    {
        private $capacity = "";
        private $price = 0;

        #konstruktor
        public function __construct($capacity = "", $price = 0){
            $this->capacity = $capacity;
            $this->price = $price;
        }

        #prosedur setter
        public function setCapacity($capacity){
            $this->capacity = $capacity;
        }

        public function setPrice($price){
            $this->price = $price;
        }

        #prosedur getter
        public function getCapacity(){
            return $this->capacity;
        }

        public function getPrice(){
            return $this->price;
        }

        public function __destruct(){ }

    }
?>